### ISSUE TITLE
The syntactic structure of PPs in Det function

### ISSUE THREAD

**Issue by BrettRey** [nert-nlp/cgel#71]
In _over thirty dollars_, is _thirty_ a fused DetHead?
What about _at least thirty dollars_?

**Comment by nschneid** [comment]
p. 432 refers to "at least" as a PP within the DP. We can treat "least" as a fused Mod-Head (cf. "at best").

p. 433, "PPs in lieu of DPs" would cover "over thirty dollars". "In lieu of", not "wrapping", suggests that we could just say the number is noun heading an NP.

**Comment by BrettRey** [comment]
I think _least_ (along with _at all_, _at most_, and _at once_) would be a fused Det-Head, not a modifier of any sort. Only in cases like _at the most_ and _at the least_ would it be a fused Mod-Head.

We could say that the number is an N, but I don't think that's right. For one thing, its number matters for agreement. Consider, also, _under approximately 30 dollars_, where it's hard to come up with another case where the _30_ in _approximately 30_ is clearly a noun. 

That leaves us with the ugliness of a fused Det-Head inside a fused Det-Head in cases like _We've got just under 30._ There's also cases like _under about 30 people_, where you have a two Det-Head:PPs.

**Comment by BrettRey** [comment]
[texstudio_BMOOpX.pdf](https://github.com/nert-nlp/cgel/files/11065498/texstudio_BMOOpX.pdf)

**Comment by nschneid** [comment]
I don't understand why "about 30" should be Det-Head. Why not "under" taking a PP complement?

**Comment by BrettRey** [comment]
I don't think _under_ takes _about_ PP complements in general, does it? *_I hid it under about the pillows_

**Comment by BrettRey** [comment]
With agreement, it would be similiar to 

1. [_More_/_Less_/_Fewer_ than twenty] people came to the meeting._
2. [_More_/_Less_/_Fewer_ than one] person came to the meeting._

**Comment by nschneid** [comment]
No, but *take* doesn't take PP complements in general either, yet it licenses the measurement ones (_It took under an hour_).

**Comment by BrettRey** [comment]
I don't think the _between_ argument in the **PPs in lieu of DPs** section on p. 433 is so clear as they suggest by saying "there can be no question." Consider _between the predicted and the measured strains_. And the idea that _from_ takes a _to_ PP complement (or that _to_ takes a _from_ PP complement) also seems fishy.

**Comment by BrettRey** [comment]
The complement with _take_ is an NP. _Under_ is inside the NP somewhere.

**Comment by BrettRey** [comment]
Actually, I guess my _between_ examples isn't relevant as it can't function as a subject.

**Comment by nschneid** [comment]
You have "least" in "at least" as an Adj at https://simple.wiktionary.org/wiki/least (but think that may be an error).

"The most", "the least", etc. is discussed at p. 395. Adjectival "least" is discussed on p. 1165, noting that it is the opposite of "greatest". I suppose the opposite of "at least" is "at most" rather than "at greatest", so that could be a reason to call it a determinative.

**Comment by BrettRey** [comment]
Right! _Lesser_ & _least_, with no base form. I should trust my past self with Wiktionary.

**Comment by BrettRey** [comment]
So adjective _least_ should be paraphrasable by _smallest_ or _slightest_. I don't think _at smallest_ or _at slightest_ works, and so, in analogy with _at most_, I think _at least_ has a D.

**Comment by nschneid** [comment]
The situation regarding Det-Head fusion is complicated by the necessity of "the" in certain situations:

- Kim has 15 dollars and Jim has a thousand cents. Who has more money and who has **less** (money)?
- Kim has 15 dollars, Jim has a thousand cents, and Nim has a dozen bucks. Who has the most money and who has **the least** (money)?

p. 395: "We analyze this *the* as modifying the comparative quantifier", so :Det(-Head) [DP :Mod [D the] :Head [D least]].

I suppose calling *the* a modifier suggests that *least* might serve as Det-Head even without *the*.

What do you make of the comparison with other *at* expressions:

Expression | Adjunct | Det function | Omitted nominal
-----------|---------|--------------|-----------------
[i] *at most/least* | It'll cost 5 dollars at least. | It'll cost [at least 5] dollars. | It'll cost several dollars—at least 5.
[ii] *at best/worst* | It'll cost 5 dollars at best. | *It'll cost [at best 5] dollars. | ?It'll cost several dollars—at best 5.
[iii] *at (a) maximum/minimum* | It'll cost 5 dollars at minimum. | *It'll cost [at minimum 5] dollars. | *It'll cost several dollars—at minimum 5.

**Comment by BrettRey** [comment]
This use of _the_ seems fine to me. Small change: 
:Det(-Head) [DP :Mod **[DP :Head** [D the]] :Head [D least]]

The comparison with other at expressions also seems to me to point to D.

**Comment by nschneid** [comment]
How would you analyze "at least an hour"?

**Comment by BrettRey** [comment]
![image](https://user-images.githubusercontent.com/1271077/227733637-17ae7529-76b4-45f9-a907-6d9852fc4534.png)

**Comment by nschneid** [comment]
So if I have it straight, the following structures (leaving off Nom nodes for brevity):

Measure expression | Measure + article | Measure + numeral
--------------------|-------------------------|--------------------------
approximately | [NP :Mod [AdvP approximately] :Head [an hour]] | [NP :Det [DP :Mod [AdvP approximately] :Head [D 5]] :Head [hours]]
at least | [NP :Mod [PP at :Obj [NP :Det-Head [DP least]]] :Head [an hour]]   | [NP :Det [DP :Mod [PP at :Obj [NP :Det-Head [DP least]]] :Head [D 5]] :Head [hours]]
about | [PP about :Obj [an hour]] | [NP :Det [PP about :Obj [NP :Det-Head [DP 5]]] :Head [hours]]

**Comment by BrettRey** [comment]
Yes, and I have no problems with the first two rows, but I'm bitter about the third, specifically the rightmost cell.

**Comment by nschneid** [comment]
I'm unsure about the whole thing. :D

I take it the argument that approximators sometimes belong to a DP, rather than as a peripheral modifier, is strongest where the noun is not a unit of measure, like "approximately 5 cookies" (*approximately cookies).

For me, "**At the very most twenty** people agreed to help" (p. 432) sounds forced. I am inclined to interpret "at the very most" as a supplement to "twenty people", and "at least" as a fixed expression in this usage before a counted noun that is not a unit of measure. But that may be a point of variation.

The idea that "about an hour" would be a PP occurring where an NP would normally be expected definitely bothers me. This is described on p. 646:

![image](https://user-images.githubusercontent.com/985263/227735837-feca9f81-7f91-4b23-9d8b-83ff7d6c9081.png)

By the way, it is not just in object function, but also cases like "about an hour later" or "about an hour after we left" or "about a mile away". Apart from approximators, the extent is typically not expressed with a PP.

It's as if the PP shell is invisible to the licensing phrase. An analogy can be made to head nouns that are transparent w.r.t. number: "[A bunch [of cookies]] are...". But there, at least we're talking about two NPs, not a PP that behaves externally like an NP.

**Comment by nschneid** [comment]
They sat on a stool.
They sat right on a stool.
the stool they sat (right) on
the stool on which they sat

They spent a year here.
the year they spent here

They spent about a year here.
They spent just about a year here.
*the year they spent (just) about __ here
*the year about which they spent here

**Comment by nschneid** [comment]
I guess I would feel more comfortable calling "about" an intransitive PP peripheral modifier within the NP. This construction could be explained as having grammaticalized from reanalysis in a predicative context (easier to see with "over" or "under" than "about"):

The duration of the event exceeded an hour.
The duration of the event was [PP over [an hour]]. (duration > an hour)
The duration of the event was [NP over an hour]. (duration = over an hour)

**Comment by BrettRey** [comment]
What do we do with this: _About the most exotic thing he did was give blood every few months or so_

**Comment by BrettRey** [comment]
https://www.cambridge.org/core/journals/journal-of-linguistics/article/abs/prepositional-phrases-as-subjects-and-objects1/DE0F8683907FFB82DBD70B61E9E75CDD

**Comment by nschneid** [comment]
This came up in treebanking today. Here are trees based on the above analysis, which seems to be what we are currently doing in CGELBank, though I don't love it.

[approx.pdf](https://github.com/user-attachments/files/15895110/approx.pdf)

**Comment by BrettRey** [comment]
Me neither, for what that's worth.
